src/test/kotlin/uk/gov/dluhc/printapi/database/repository/CertificateRepositoryIntegrationTest.kt

```diff
@@ -329,23 +351,4 @@ internal class CertificateRepositoryIntegrationTest : IntegrationTest() {
             assertThat(actual).isEqualTo(2)
         }
     }
-
-    private fun assertCertificateRecursiveEqual(actual: Certificate, expected: Certificate) {
```

Comment:
This was really complicated and I felt could be much more succinctly replaced with
```
            assertThat(actual)
                .usingRecursiveComparison()
                .ignoringFieldsMatchingRegexes(*IGNORED_FIELDS)
                .isEqualTo(expected)
```

Reply:
I remember being frustrated with the date comparisons not working due to rounding - glad you found ways to simplify 👍 

Nice trick:
```
Instant.now().minusSeconds(it.toLong()).truncatedTo(SECONDS)
```